.github/workflows/ci.yaml

```diff
@@ -82,7 +82,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --no-fail-fast --all
+          args: --no-fail-fast --all -- --test-threads=1
```

Comment:
Just to make sure, do we really want to break multithreaded tests?

Reply:
**stdout** tests require a single thread.
Because **stdout** is stored in **OnceCell**.

Reply:
I know the reason, I just wanted to be sure that you really want to break multithreaded execution.

Reply:
Combined tests